packages/web/e2e/uploadTrack.test.ts

```diff
@@ -0,0 +1,224 @@
+import { expect } from '@playwright/test'
+import {
+  EditTrackPage,
+  UploadFinishPage,
+  UploadSelectPage
+} from './page-object-models/upload'
+import {
+  RemixSettingsModal,
+  AccessAndSaleModal,
+  AttributionModal,
+  StemsAndDownloadsModal
+} from './page-object-models/modals'
+import { test } from './test'
+
+test('should upload a remix, hidden, AI-attributed track', async ({ page }) => {
```

Comment:
I like how many cases you covered! Are we going to have "essential" and "comprehensive" suites like @DejayJD set up in https://github.com/AudiusProject/audius-protocol/pull/7859

Would love to discuss which tests belong in which suite

Reply:
we definitely can! Maybe we should define a target runtime for tests - personally I think if the tests take < 10 min there's no need to split them up but I'm curious what others think is tolerable

Reply:
That does seem reasonable! I'm more so concerned with the maintainability and size of our core test suite. I've been in situations where the test suite is big and always broken and I want to avoid that

IMO core test suite should only be the super critical paths:
* Sign up / sign in
* Stream track
* Upload track

And everything else could be in the "comprehensive" suite which is less blocking and we have more time to fix. BUT who knows, if all of our tests are written as well as these maybe we don't have much of a maintenance concern :)

Reply:
yea I would love to revisit that concept at some point in the future. Grep filtering is already built into playwright so we can quickly port over that PR. Although I do want to let our pw stuff run for a bit and see how fast/flaky things are out in the wild